specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-08-15-preview/cosmos-db.json

```diff
@@ -10587,6 +10587,10 @@
           "type": "string",
           "format": "date-time",
           "description": "Time to which the account has to be restored (ISO-8601 format)."
+        },
+        "sourceBackupLocation": {
```

Comment:
Looks like this RestoreParameterBase is also being used by InAccount restore, meaning if a collection or database was in-account restored, then it will have this parameter as well.